code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -252,6 +252,40 @@ impl assets_registry::Config for Runtime {
 	type WeightInfo = weights::assets_registry::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub PabloPalletId: PalletId = PalletId(*b"pal_pblo");
+	pub TWAPInterval: u64 = (MILLISECS_PER_BLOCK as u64) * 10;
+	pub MaxStakingRewardsPools: u32 = 10;
+	pub MaxRewardConfigsPerPool: u32 = 10;
+	pub MaxStakingDurationPresets: u32 = 10;
+	pub PbloAssetId: CurrencyId = CurrencyId::PBLO;
+	pub MsPerBlock: u32 = MILLISECS_PER_BLOCK;
+}
+
+impl pablo::Config for Runtime {
```

Comment:
Let's not use PIcasso for this test. We have to remove as any Picasso modification needs to be separately approved as of recent agreements.